iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/fileio/FileIOManager.java

```diff
@@ -33,9 +36,15 @@ public class FileIOManager {
   @Qualifier("LocalFileIO")
   FileIO localFileIO;
 
+  @Autowired(required = false)
```

Comment:
```suggestion
  @Qualifier("ADLSFileIO")
  @Autowired(required = false)
```
Do we need qualifier here?

Reply:
Probably not. I will remove

Reply:
Sushant said we don't because we only have one storage type using `ADLSFileIO` so it Autowires directly. See this comment: [https://github.com/linkedin/openhouse/pull/120#discussion_r1635257327](https://github.com/linkedin/openhouse/pull/120#discussion_r1635257327)